### ISSUE TITLE
For you screen Intro accessibility issues

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#432]
Related to #239 

For you screen (Intro)
1. The focus should go to the "For you" heading and then go to More options. (It can be tested by activating voiceover on For you page).
2. On the Following screen, select subjects should be labelled as button and it should highlight the whole button instead if Select subjects.
3. On the Saved screen, Go to publications should be labelled as a button as well and should be highlighted as a whole and labelled as a button
